README.md

```diff
@@ -61,14 +61,7 @@ const stripe = await useClientStripe()
 1. Add `@unlok-co/nuxt-stripe` dependency to your project
```

Comment:
@danielroe I would add more detail on what is going to happen in my project. wdyt? I'm doing the same on leaflet project as I was playing with. Will ping you there as well.
```suggestion
Run `npx` command to add `@unlok-co/nuxt-stripe` into your `package.json` and `nuxt.config.ts`
```